tests/snapshots/tests.contrib.rq.test_rq.test_enqueue_many.json

```diff
@@ -0,0 +1,228 @@
+[[
+  {
+    "name": "rq.queue.enqueue_job",
+    "service": "rq",
+    "resource": "tests.contrib.rq.jobs.job_add1",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
```

Comment:
So we'd expect 3 spans with parent_id 0 for each job that we enqueue, and then the children for each of them while running them through the worker. That makes sense to me I think.